.github/workflows/build.yml

```diff
@@ -0,0 +1,110 @@
+# This pipeline builds the sfpowerscripts plugin
+
+
+name: 'sfpowerscripts Build Pipeline'
+
+on:
+  push:
+    branches:
+      - develop
+    paths-ignore:
+      - 'docs/**'
+
+  workflow_dispatch:
+
+
+jobs:
+
+#Merge to Develop, Deploy Alpha builds
+  alpha:
+    name: 'Alpha Builds'
+    runs-on: ubuntu-latest
+    if: contains(github.ref,'refs/heads/develop')
+    environment:
+      name: alpha
+    concurrency: alpha
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: BuildCLI (sfdx-plugin)
+        uses: ./.github/workflow-templates/buildcli.yml
+        timeout-minutes: 20
+        with:
+          commitToGit: true
+          version: 'alpha'
+          publish: true
+
+
+
+# Beta Stage
+  beta:
+    name: 'Beta Builds'
+    runs-on: ubuntu-latest
+    if: contains(github.ref,'refs/heads/develop')
+    needs: alpha
+    environment:
+      name: beta
+    concurrency: beta
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: BuildCLI (sfdx-plugin)
+        uses: ./.github/workflow-templates/buildcli.yml
```

Comment:
Let's diverge from the original pipeline design here, and use the `promote` workflow instead of relying on conditionals within the `build` workflow to only execute steps relating to promotion.

Reply:
The `promote` reusable workflow should have two jobs: `beta` and `prod`

Reply:
in progress